MatrixSDK/Data/Store/MXFileStore/MXFileStore.m

```diff
@@ -293,6 +293,27 @@ - (void)diskUsageWithBlock:(void (^)(NSUInteger))block
     });
 }
 
+- (void)logFiles
+{
+    NSLog(@"[MXFileStore] logFiles: Files in %@:", self->storePath);
+    NSArray *contents = [[NSFileManager defaultManager] subpathsOfDirectoryAtPath:self->storePath error:nil];
+    NSEnumerator *contentsEnumurator = [contents objectEnumerator];
+    
+    NSUInteger fileCount = 0, diskUsage = 0;
+
+    NSString *file;
+    while (file = [contentsEnumurator nextObject])
+    {
+        NSDictionary *fileAttributes = [[NSFileManager defaultManager] attributesOfItemAtPath:[self->storePath stringByAppendingPathComponent:file] error:nil];
+        NSUInteger fileSize = [[fileAttributes objectForKey:NSFileSize] intValue];
```

Comment:
`NSFileSize` working for folders?

Reply:
weirdly, yes :)

Reply:
Seems that’s why the total size doesn’t match but, anyway i’m ok if you’re aware.